platform/ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -11,7 +12,7 @@ const ContextMenu = ({ items }) => {
       {items.map((item, index) => (
```

Comment:
@wayfarer3130 This line produces exception when `items` is undefined.
I would suggest to change `items.map....` to `(items || []).map...`

Reply:
Added a specific check for undefined.